extensions/WSLExtension/DevHomeProviders/WslProvider.cs

```diff
@@ -97,4 +113,25 @@ public IAsyncOperation<ComputeSystemsResult> GetComputeSystemsAsync(IDeveloperId
         var notImplementedException = new NotImplementedException($"Method not implemented by WSL Compute System Provider");
         return new ComputeSystemAdaptiveCardResult(notImplementedException, notImplementedException.Message, notImplementedException.Message);
     }
+
+    /// <summary>
+    /// Gets a boolean indicating whether the Virtual Machine Platform feature is enabled.
+    /// </summary>
+    /// <returns>True only when the feature is enabled. False otherwise.</returns>
+    private bool IsVirtualMachinePlatformFeatureEnabled()
+    {
+        var searcher = new ManagementObjectSearcher($"SELECT InstallState FROM Win32_OptionalFeature WHERE Name = 'VirtualMachinePlatform'");
+        var collection = searcher.Get();
+
+        foreach (var instance in collection)

```

Comment:
This `foreach` will always return on first iteration, is this right? Consider a `FirstOrDefault()` if we are interested in the first item only.